src/modules/endpoint/__tests__/DefaultDataContainer.spec.js

```diff
@@ -17,6 +28,118 @@ const testCity = new CityModel({
   prefix: 'Stadt'
 })
 
+const testLanguage = new LanguageModel({
```

Comment:
I added yesterday a few builders for test data: https://github.com/Integreat/integreat-react-native-app/commit/e66f4fa0d1ed9be7793489030b734a55a7d13caa

Would be cool if you can use them here too :)

Reply:
If they are fine for you, you can just cherry-pick that commit

Reply:
Looks good, I will use it